__tests__/storage.test.ts

```diff
@@ -1,6 +1,6 @@
 import * as esCookie from 'es-cookie';
 import { MockedObject } from 'ts-jest/dist/utils/testing';
-import { mocked } from 'ts-jest/utils';
+import { mocked } from 'jest-mock';
```

Comment:
Is it an issue that `jest-mock` isn't defined in devDeps so would be a transient module? Alternatively, it looks like `mocked` is available on the [global jest object](https://jestjs.io/docs/27.x/jest-object#jestmockedtitem-t-deep--false) directly so maybe the import isn't needed?

Reply:
Updated, thanks!